app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -66,6 +66,8 @@ def start
     end
 
     queue_signal(:pod_wait)
+  rescue
+    queue_abort("unexpected exception, see log")
```

Comment:
I think you should rescue the exception and pass that into the `queue_abort` method.  That way it can log the specific exception that caused it to abort.  Like:

```ruby
rescue => e
  queue_abort("unexpected exception, see log", e)
end
```